_source/_includes/metric-shipping/telegraf-setup-win.md

```diff
@@ -0,0 +1,5 @@
+1. Navigate to the [Influxdata portal](https://portal.influxdata.com/downloads/).
+2. Select the latest version of Telegraf in the **Version** menu.
+3. Select **Windows Binaries 64-bit** in the **Platform** menu.
+3. Copy and paste the wget link into your browser address bar. For example, https://dl.influxdata.com/telegraf/releases/telegraf-1.19.2_windows_amd64.zip.
+4. Save the archive and extract its contents into `C:\Program Files\InfluxData\telegraf\`.
```

Comment:
Can be changed to:
wget https://dl.influxdata.com/telegraf/releases/telegraf-1.19.2_windows_amd64.zip